0010-Archivematica-original-design-principles.md

```diff
@@ -0,0 +1,49 @@
+# Archivematica original design principles
+
+* Status: accepted
+
+## Authors
+
+* Peter Van Garderen
+* Evelyn McLellan
+
+## Context and problem statement
+
+There is no open-source software system capable of implementing the functional requirements of the OAIS reference model (ISO 14721:2003). Digital preservation specialists must use multiple tools, which can be difficult to install and use, to perform discrete preservation actions. Those tools produce metadata that do not conform to digital preservation and exchange standards and schemas, and do not provide a way to automatically generate standardized, system-independent, self-documenting Archival Information Packages (AIPs) that package content and Preservation Description Information (PDI) as described by OAIS. Repository applications such as Fedora are capable of performing some but not all OAIS preservation actions, and tend to be complex to develop and maintain, posing a risk to future retrieval and readability of the AIPs. In fact, any middleware repository or database software that is required to access and read AIPs is inherently a risk to their long-term usability.
+
+## Decision drivers
+
+Artefactual designed an open-source, web-based archival description and access system called ICA-AtoM (Access To Memory) that has a broad user base around the world. ICA-AtoM does not provide digital preservation functionality as described by OAIS. It would benefit ICA-AtoM users to be able to integrate with a back-end system designed to preserve digital objects that are linked to access copies in ICA-AtoM. The system should also be usable on its own or in conjunction with other access tools.
+
+## Considered options
+
+1. Develop a repository interface to a digital object store (e.g. Fedora or DSpace); or
+2. Develop a database-backed application to manage AIPs (e.g. Preservica); or
+3. Use a micro-services approach with loosely-coupled tools to provide digital curation services built around file system storage (e.g. University of California’s Merritt System).
+
+
```

Comment:
Peter, the linter is reporting some minor issues like this extra line being reported as:

    MD012/no-multiple-blanks Multiple

It's not a big deal though, we can fix some other time.

Reply:
thanks